frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FormCreator.tsx

```diff
@@ -1,7 +1,10 @@
 import React, { useState, useEffect, useMemo } from "react";
+import { jt, t } from "ttag";
 import _ from "underscore";
 
+import ExternalLink from "metabase/core/components/ExternalLink";
 import { ActionForm } from "metabase/actions/components/ActionForm";
+import SidebarContent from "metabase/query_builder/components/SidebarContent";
```

Comment:
nit: are we ok with importing something from the query builder here? maybe move it to some common component? e.g. `metabase/core/components` or `metabase/common/components`

Reply:
^ the above isn't a blocker to merge the PR